rolling/release-build.yaml

```diff
@@ -16,7 +16,7 @@ notifications:
   - ros2-buildfarm-rolling@googlegroups.com
   maintainers: true
 sync:
-  package_count: 500
+  package_count: 499
```

Comment:
I think leaving it at 500, along with the required `desktop` package, is fine.  We could also consider using the percentages now that that is a feature in `ros_buildfarm`, but I don't feel strongly about it.

Reply:
I got ahead of myself and merged this with the decremented package count :facepalm: Because it's only off by one, and because I expect that we'll want to relax this even further for at least the first sync to testing. I'm not going to make any further changes just yet.